commands/main_test.go

```diff
@@ -12,12 +12,15 @@ import (
 func TestRequiresDaemon(t *testing.T) {
 	tf.UnitTest(t)
 
-	reqWithDaemon, err := cmds.NewRequest(context.Background(), []string{}, nil, []string{"chain"}, nil, chainCmd)
+	reqWithDaemon, err := cmds.NewRequest(context.Background(), []string{"chain", "head"}, nil, []string{}, nil, rootCmdDaemon)
```

Comment:
These changes are required because previously we were passing `chain", "head` as arguments, which this are not, they go in path ([relevant method signature](https://github.com/ipfs/go-ipfs-cmds/blob/746d639be4ddb96c76ad8a0020466b069edb5e90/request.go#L27)). Your modification addresses this. Is that correct?